requirements.txt

```diff
@@ -4,10 +4,14 @@
 sphinx==5.0.0
 sphinx-gallery==0.11.1
 sphinx_design
+nbsphinx
 docutils==0.16
 sphinx-copybutton
-tqdm==4.66.1
-numpy==1.24.4
+pypandoc==1.12
+pandocfilters
+markdown
+tqdm
+numpy
```

Comment:
Please do not unpin numpy and tqdm, but rather specify versions that do work for you (all those pins were added when updates broke tutorials)